experimenter/tests/integration/nimbus/utils/filter_expression.js

```diff
@@ -23,10 +23,11 @@ async function remoteSettings(arguments) {
             throw ex;
         }
     }
-    const ExperimentManager = ChromeUtils.importESModule("resource://nimbus/lib/ExperimentManager.sys.mjs");
+    const { ExperimentAPI } = ChromeUtils.importESModule("resource://nimbus/ExperimentAPI.sys.mjs");
     const TargetingContext = ChromeUtils.importESModule("resource://messaging-system/targeting/Targeting.sys.mjs");
 
     const _experiment = JSON.parse(arguments[1]);
+    ExperimentAPI.ready();
```

Comment:
This returns a promise
```suggestion
    await ExperimentAPI.ready();
```